src/pages/LogIn/components/LoginForm.tsx

```diff
@@ -10,18 +10,20 @@ import Client, { User } from "@fnndsc/chrisapi";
 
 const LoginFormComponent = () => {
   const history = useHistory();
-  const [showHelperText] = useState(false);
-  const [usernameValue, setUsernameValue] = useState('');
-  const [isValidUsername] = useState(true);
-  const [passwordValue, setPasswordValue] = useState('');
-  const [RememberMeClick, setRememberMeClick] = useState(true);
+  const [helperText, setHelperText] = useState("");
```

Comment:
It seems to me that the use of a state for a `helperText` is unnecessary.

Have you considered making the `helperText` attribute in `LoginForm` to be "Invalid username or password." and then using `!isValidCredentials` as the boolean flag for `showHelperText`?

Reply:
That is a good point. The reason why I had it this way was because `helperText` is actually a general purpose help message, not strictly just for errors. By default, it displays in black text, but if `isValidUsername` or `isValidPassword` is false, it'll be styled red. My initial thought was to keep the helper text as generalized as possible in case we ever have the need to display messages other than "Invalid username or password.". Thinking about it now though, I can't really think of any other messages we'd ever want to display for our application. Do you think it's necessary to keep it general like this, or no?

Reply:
@boris-nguyen yeah, I agree. I'd say that an error message is already quite generic in itself, as it's error text that we can show for any variety of log-in issues, so I think we could do with a specific usage of `errorText` would be more than sufficient

Reply:
@janakitti In my personal opinion, it is okay with setting the `helperText` to constant text because a) currently there is no use case for other messages and based on what ChRIS backend responds during authentication, I don't think there would be other use cases b) The PatternFly Login component is already generic so if in the future we do want to add more messages, it will be quite a quick code change.

Reply:
Ok, agreed! Suggestion has been implemented: https://github.com/darwinai/covidnet_ui/pull/30/commits/29d4fcb4fccce5ee0818d56addbf5f29e0af3611